test/api-helper/project/GitHubLazyProjectLoader.test.ts

```diff
@@ -25,26 +25,34 @@ import {
 import * as assert from "power-assert";
 import { save } from "../../../lib/api-helper/project/CachingProjectLoader";
 import { CloningProjectLoader } from "../../../lib/api-helper/project/cloningProjectLoader";
-import { LazyProjectLoader } from "../../../lib/api-helper/project/LazyProjectLoader";
+import { GitHubLazyProjectLoader } from "../../../lib/api-helper/project/GitHubLazyProjectLoader";
 import { SingleProjectLoader } from "../../../lib/api-helper/testsupport/SingleProjectLoader";
 
 const credentials = {
     token: process.env.GITHUB_TOKEN,
 };
 
-describe("LazyProjectLoader", () => {
+describe("GitHubLazyProjectLoader", () => {
+
+    it("should ", async () => {
+        const id = new GitHubRepoRef("this.is.invalid", "nonsense");
+        const lpl = new GitHubLazyProjectLoader(CloningProjectLoader);
+        const p: Project = await save(lpl, { credentials, id, readOnly: false });
+        assert.strictEqual((p as any).materialized(), false);
+        assert((p as any).materialized);
```

Comment:
This test to see if the property is truthy seems superfluous since you call the function of the property in the previous line.

Reply:
It does now. yeah. Fixing.